nylas/client/client.py

```diff
@@ -104,11 +104,16 @@ def access_token(self, value):
             if 'Authorization' in self.session.headers:
                 del self.session.headers['Authorization']
 
-    def authentication_url(self, redirect_uri, login_hint='', state=''):
+    def authentication_url(self, redirect_uri, login_hint='', state='', scopes=None):
```

Comment:
```suggestion
    def authentication_url(self, redirect_uri, login_hint='', state='', scopes=('email', 'calendar', 'contacts')):
```

Reply:
nitpick: I think it might be better to put the default in the parameters so people know what it defaults to reading the code